README.md

```diff
@@ -136,6 +136,76 @@ custom:
   datadog:
     enabled: ${self:custom.staged.dd_enabled, true}
 ```
+### Serverless Monitors 
+
+There are seven recommended monitors with default values pre-configured. 
+
+| Monitor              | Metrics                                                                                   | Default Threshold | Serverless Monitor ID |
```

Comment:
Since the threshold cannot be customized, I would call it the "threshold" rather than the "default threshold"